### ISSUE TITLE
Not able to download Mac OS version from the homepage.

### ISSUE THREAD

**Issue by ujeshkrishnan** [marktext/marktext#3066]
### Description

- [ ] Can you reproduce the issue? 

### Steps to reproduce

1. [First step]
2. [Second step]
3. [and so on...]

**Expected behavior:**

**Actual behavior:**

**Link to an example: [optional]**

### Versions

- MarkText version:
- Operating system:

**Comment by DYH-Hong** [comment]
Yep. It should have directed you to `marktext-x64.dmg`, not `marktext.dmg`.

**Comment by ujeshkrishnan** [comment]
But it is pointed to a broken page now. The button is not working.

**Comment by laggage** [comment]
I think this issue should belong to the [MarkText/website](https://github.com/marktext/website.git) repository, so I just created a new issue in that repository and submitted some code to repair the problem by the way

**Comment by Jocs** [comment]
The new official website is already available, https://www.marktext.cc/
